polaris-service/src/test/java/org/apache/polaris/service/test/PolarisConnectionExtension.java

```diff
@@ -109,10 +118,40 @@ public void beforeAll(ExtensionContext extensionContext) throws Exception {
     }
   }
 
+  @Override
+  public void afterAll(ExtensionContext context) {
+    if (metaStoreManagerFactory instanceof LocalPolarisMetaStoreManagerFactory) {
+      metaStoreManagerFactory.purgeRealms(List.of(realm));
+    }
+  }
+
   public static String getTestRealm(Class testClassName) {
     return testClassName.getName().replace('.', '_');
   }
 
+  public static void createTestDir(String realm) throws IOException {
+    // Set up the database location
+    Path testDir = Path.of("build/test_data/iceberg/" + realm);
```

Comment:
iceberg?

Reply:
Yeah. That is like this currently. Let me change to polaris.